tests/phpunit/tests/l10n/wpWordCount.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/**
+ * @group l10n
+ * @group i18n
+ *
+ * @covers ::wp_word_count
+ */
+class Tests_L10n_wpWordcount extends WP_UnitTestCase {
+	protected static $settings;
+
+	public static function wpSetUpBeforeClass() {
+		self::$settings = array(
+			'shortcodes' => array( 'shortcode' ),
+		);
+	}
+
+	/**
+	 * @ticket 57987
+	 *
+	 * @dataProvider data_get_string_variations
+	 *
+	 * @param string $text     Text to count elements in.
+	 * @param int    $expected Expected word count.
+	 */
+	public function test_wp_word_count_should_return_the_number_of_words( $text, $expected ) {
+		$this->assertSame( wp_word_count( $text, 'words', self::$settings ), $expected['words'] );
+	}
+
+	/**
+	 * @ticket 57987
+	 *
+	 * @dataProvider data_get_string_variations
+	 *
+	 * @param string $text     Text to count elements in.
+	 * @param int    $expected Expected character count.
+	 */
+	public function test_wp_word_count_should_return_the_number_of_characters_excluding_spaces( $text, $expected ) {
+		$this->assertSame( wp_word_count( $text, 'characters_excluding_spaces', self::$settings ), $expected['characters_excluding_spaces'] );
+	}
+
+	/**
+	 * @ticket 57987
+	 *
+	 * @dataProvider data_get_string_variations
+	 *
+	 * @param string $text     Text to count elements in.
+	 * @param int    $expected Expected character count.
+	 */
+	public function test_wp_word_count_should_return_the_number_of_characters_including_spaces( $text, $expected ) {
+		$this->assertSame( wp_word_count( $text, 'characters_including_spaces', self::$settings ), $expected['characters_including_spaces'] );
+	}
+
+	/**
+	 * @ticket 56698
```

Comment:
```suggestion
	 * @ticket 56697
```
:1234: Applies to later tests too.

Reply:
fixed by 85886e8
The correct ticket number is 57987 😅